src/DataManagement/RecurringUpdater.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using ePiggyWeb.DataBase;
+using ePiggyWeb.DataManagement.Entries;
+using ePiggyWeb.Utilities;
+
+namespace ePiggyWeb.DataManagement
+{
+    public static class RecurringUpdater
+    {
+        public static bool UpdateRecurring(IEntryManager entryManager)
+        {
+            foreach (var entry in entryManager.EntryList.GetBy(true))
+            {
+                var differenceInMonths = TimeManager.DifferenceInMonths(laterTime: DateTime.Today, earlierTime: entry.Date);
```

Comment:
As lecturer mentioned we can probably use just `laterDate - earlierDate`. Atleast I think

Reply:
I tried this, however laterDate - earlierDate returns a time span, which cannot be converted to months, which we are using here